Library/Homebrew/extend/os/mac/requirements/java_requirement.rb

```diff
@@ -1,13 +1,30 @@
 class JavaRequirement < Requirement
-  cask "java"
```

Comment:
The `@cask` doesn't seem to be set any more, though. Does the cask message still get output as expected now? If so: 🆒 and ignore me.

Reply:
The base `Requirement` class has:
```
class Requirement
  include Dependable

  attr_reader :tags, :name, :cask, :download, :default_formula

  def initialize(tags = [])
    @default_formula = self.class.default_formula
    @cask ||= self.class.cask
    @download ||= self.class.download
    @formula = nil
    tags.each do |tag|
      next unless tag.is_a? Hash
      @cask ||= tag[:cask]
      @download ||= tag[:download]
    end
    @tags = tags
    @tags << :build if self.class.build
    @name ||= infer_name
  end
```

Reply:
:cool:. If it works and you're happy with it: 🚢